tests/lib/repack-kernel.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
```

Comment:
ideally $SOMEDAY this whole script becomes a command on the ubuntu-core-initramfs snap